### PR TITLE
Geno 3-2 and 3-3

### PR DESCRIPTION
**Pull request by contingency40**
* Titanoboa's boss fights
* Arachne's encounters
* Arachne's boss fight
* M.O.W. dialogue
* Sphinx battle
* Misc translation fixes
* Item text
* Gargoyle dialogue
* Micah encounter

### CODE REVIEW

File: Gaijinizer patch/Map003.txt [hunk 1]
```diff
@@ -3042,7 +3047,9 @@ Moderately good, good mind#MTLed
 > CONTEXT: Map003/events/10/pages/2/210/Dialogue
 > CONTEXT: Map166/events/10/pages/2/210/Dialogue
 > CONTEXT: Map195/events/10/pages/2/210/Dialogue
-Heheheheheh!!\\C[10]Arachne\\C[0] The\\C[10] Killed\\C[0] Look at it!#MTLed
+Kukukukuku!!
+
+You \\C[10]squashed \\C[0]that \\C[10]Arachne \\C[0]bug!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map003.txt:3053]: Note

File: Gaijinizer patch/Map097.txt [hunk 2]
```diff
@@ -1269,8 +1269,9 @@ Take it there.#MTLed
 > CONTEXT: Map097/events/2/pages/1/113/Dialogue/enemy10
 > CONTEXT: Map097/events/6/pages/1/113/Dialogue/enemy10
 > CONTEXT: Map097/events/8/pages/1/113/Dialogue/enemy10
-Protect this land...a proud beast.
-If you go ahead from here, I ask you .........#MTLed
+I'm a proud, legendary beast who safeguards this
+land. If you wish to travel further, you must
+first answer thus...#MTLed
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map097.txt:1275]: Add attributation here

File: Gaijinizer patch/Items.txt [hunk 3]
```diff
@@ -1083,7 +1083,7 @@ It exudes a dark aura creating an unpleasant mood.#afrgPR
 > BEGIN STRING
 宝珠『絶望』
 > CONTEXT: Items/110/name
-Orb "Despair"#afrg
+Orb of Despair#afrg
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Items.txt:1087]: Might want to double check the description for this item, it's a bit weak.

File: Gaijinizer patch/Map097.txt [hunk 4]
```diff
@@ -1303,9 +1306,10 @@ You will be transmitted to me.........#MTLed
...
 > CONTEXT: Map097/events/6/pages/1/126/Dialogue/kazuya-3
 > CONTEXT: Map097/events/8/pages/1/126/Dialogue/kazuya-3
-What's .... You know. I'm rushing ahead
-If you don't want to withdraw, let's fight soon
-...!#MTLed
+Huh, so you really do get it.
+
+I'm in a hurry so if you plan on fighting, let's
+get this done already!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map097.txt:1313]: Comma after "hurry".

File: Gaijinizer patch/Map132.txt [hunk 5]
```diff
@@ -251,60 +251,67 @@ Hyakki...#Ron
...
 決して誰にも『介入』されない……
 
 僕だけの『力』だ……
 > CONTEXT: Map132/events/1/pages/1/143/Dialogue/kazuya-3
-I will never "intervene" by anyone ...
-It is my only "power" ...#MTLed
+No one will intervene...
+
+This is my power alone...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map132.txt:315]: Note

File: Gaijinizer patch/Map136.txt [hunk 6]
```diff
@@ -178,8 +185,10 @@ p.s.
 もしこの世界に不満なら、やり直してもええんやで……
 > CONTEXT: Map136/events/5/pages/1/46/Dialogue
 > CONTEXT: Map190/events/4/pages/1/46/Dialogue
-p.s. I wonder if you die early but...If you're not happy
-with this world, it's okay to try again ...#MTLed
+P.S. I'd rather you died sooner than later...
+
+But if by chance you're not happy with this world, it's
+okay to try again...#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map136.txt:192]: Note

File: Gaijinizer patch/Troops.txt [hunk 7]
```diff
@@ -8369,205 +8369,211 @@ Sphinx (G)#MTLed
...
-...... but after a while. I stood up with a shaking leg ...#MTLed
+But after a moment, she gets back up on unsteady legs...#Ron
 > END STRING
 
 > BEGIN STRING
 い……いたい……
 > CONTEXT: Troops/210/pages/2/15/Dialogue/enemy11
-Want to say……#MTLed
+M-Man that really hurt...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:8387]: Comma after "Man".

File: Gaijinizer patch/CommonEvents_575.txt [hunk 8]
```diff
@@ -36,16 +40,19 @@ this is…………\\C[10] "intervention"\\C[0] .........?#MTLed
...
 > CONTEXT: CommonEvents_575/106/Dialogue/enemy17
-Kuh ......... Hehe hee......!!
-I can not think quietly...If you receive such
-Attacks in a row, you can say that ...#MTLed
+Kukukuku...!!
+It's true, I hadn't considered that...
+
+If I keep taking hits like that over and over,
+I'll...#RonProvisional
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/CommonEvents_575.txt:48]: Note

File: Gaijinizer patch/Map128.txt [hunk 9]
```diff
@@ -830,18 +834,20 @@ What is it ...#MTLed
...
 だから力も、今以上に引き出せない。
 > CONTEXT: Map128/events/4/pages/1/39/Dialogue/enemy10
-But that's your "refuge". Without being dyed, you
-can not favor evil powers. So I can not pull out
-more power than now.#MTLed
+But that's become your "escape route".
+You can't court true darkness without having
+first stained your hands. And thus, the power you
+can draw will also be limited.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map128.txt:841]: Note ("escape route")

File: Gaijinizer patch/Map128.txt [hunk 10]
```diff
@@ -870,124 +876,132 @@ I shoot this guy's head.#MTLed
...
 > BEGIN STRING
 今まで邪魔だった『くさび』が
 
  どんどん自分の体から剥がれているのを……
 > CONTEXT: Map128/events/4/pages/1/130/Dialogue/enemy10
-The "wedge" which has been a hindrance until now
-more and more being detached from my body ...#MTLed
+That "binding" that's always held you back, it's
+steadily coming off...#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map128.txt:985]: Note ("binding" -> "chains")

File: Gaijinizer patch/Map131.txt [hunk 11]
```diff
@@ -694,16 +703,17 @@ Yes...this is the last lesson ...
 > CONTEXT: Map131/events/5/pages/1/40/Dialogue/kazuya-3
 > CONTEXT: Map131/events/8/pages/1/40/Dialogue/kazuya-3
 > CONTEXT: Map131/events/9/pages/1/40/Dialogue/kazuya-3
-I'll do it ...!!! After all, you were going to
-kill you too! !#MTLed
+I'll do it...!!!
+
+I was gonna kill you either way!!#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map131.txt:709]: image

Would `Let's do it...!!!` be a valid translation here? It fits a lot better with the text.

File: Gaijinizer patch/Map131.txt [hunk 12]
```diff
@@ -651,8 +654,9 @@ exit" ...?#MTLed
 > CONTEXT: Map131/events/5/pages/1/22/Dialogue/kazuya-3
 > CONTEXT: Map131/events/8/pages/1/22/Dialogue/kazuya-3
 > CONTEXT: Map131/events/9/pages/1/22/Dialogue/kazuya-3
-I understand...I understand that I take it by
-hand...Arachne's thinking ...#MTLed
+I understand your thinking...
+
+I understand it all too well...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map131.txt:660]: Note

File: Gaijinizer patch/Map132.txt [hunk 13]
```diff
@@ -251,60 +251,67 @@ Hyakki...#Ron
...
 だけど……もう怖くない……
 
 僕は…『力』を手に入れた……
 > CONTEXT: Map132/events/1/pages/1/139/Dialogue/kazuya-3
-But...I'm not scared anymore ...
-I got..."power" ...#MTLed
+But... I'm not afraid anymore...
+
+Now that I have "power"...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map132.txt:305]: Add "this" after "have".

File: Gaijinizer patch/Map134.txt [hunk 14]
```diff
@@ -412,32 +422,34 @@ You waste everything ...#MTLed
...
+\\N[1] slowly takes the "outdated" gun from Micah's
+shaking hands...!#Ron
 > END STRING
 
 > BEGIN STRING
 あ、あぁ…………………
 > CONTEXT: Map134/events/10/pages/1/142/Dialogue/other4
-Ah...Ah .........................#MTLed
+Y-Yeah...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map134.txt:443]: Note

File: Gaijinizer patch/Map131.txt (hunk 11 above)
Comment (contingency40) [reply Gaijinizer patch/Map131.txt:709]: Something to keep in mind with "やってやる" is that final やる as a suffix is used when you're doing someone a favour, but it's a very arrogant word to use that you'd normally employ with someone you look down on. The nuance is basically, "I'll do you the favour of playing this little game of yours."

Comment (Triketos) [reply Gaijinizer patch/Map131.txt:709]: Okay, this will work better then:
```
Fine, I'll play along...!!
```

**Review by Triketos — APPROVED** [review]
Looks good after the  last few changes 👍

**Pull request merged**